.env.example

```
@@ -1,8 +1,18 @@
+DISCOVERY_LOBBY_TYPE=7    # 7 = ranked matchmaking
+DISCOVERY_MIN_MMR=3000    # minimum average MMR
+DISCOVERY_LIMIT=100       # matches per discovery query
+# DISCOVERY_PATCH=        # leave empty for latest patch
+# DISCOVERY_REGION=       # leave empty for all regions
+
+# ── Database ─────────────────────────────────────────────────────
 DB_PATH=services/ingestion/db/data.sqlite
-LOG_LEVEL=INFO
+
+# ── Logging ──────────────────────────────────────────────────────
+LOG_LEVEL=INFO
```

dotenv-linter (4.0.0)

[warning] [EndingBlankLine] No blank line at the end of the file

(EndingBlankLine)